datanode/src/test/resources/analysis/inst/cohorts/Death.json

```diff
@@ -0,0 +1,310 @@
+CREATE TABLE #Codesets (
```

Comment:
That does not look as JSON file

Reply:
Excessive test files removed to avoid confusion. Tests currently only check parsing the metadata, so these are unnecessary. In the future we need to enchance tests to run with EE and have fully working analysis samples here.